src/huggingface_hub/hf_api.py

```diff
@@ -3533,7 +3533,7 @@ def update_repo_visibility(
         token: Union[str, bool, None] = None,
         repo_type: Optional[str] = None,
     ) -> Dict[str, bool]:
-        """Update the visibility setting of a repository.
+        """(Deprecated) Update the visibility setting of a repository.
```

Comment:
```suggestion
        """Update the visibility setting of a repository.
        
        Deprecated. Use `update_repo_settings` instead.
```

Better to put it in a second line so that we can recommend to the user what they should use